$NetBSD: patch-media_CMakeLists.txt,v 1.2 2023/04/18 18:48:15 abs Exp $

Don't find "convert" or "png2icns" if they happen to be installed.

--- media/CMakeLists.txt.orig	2023-02-28 15:11:20.000000000 +0000
+++ media/CMakeLists.txt
@@ -2,8 +2,8 @@ set(LINUX_ICON_SIZES 16 22 24 32 48 64 1
 set(WINDOWS_ICON_SIZES 16 24 32 48 256)
 set(MACOS_ICON_SIZES 16 32 128 256 512)
 
-find_program(CONVERT_EXECUTABLE convert)
-find_program(PNG2ICNS_EXECUTABLE png2icns)
+# find_program(CONVERT_EXECUTABLE convert)
+# find_program(PNG2ICNS_EXECUTABLE png2icns)
 
 if(CONVERT_EXECUTABLE)
   set(ICON_FILES icons/tigervnc.ico)
